src/main/java/io/opentracing/contrib/global/GlobalTracer.java

```diff
@@ -38,43 +38,26 @@
  * The {@linkplain GlobalTracer} will not attempt to choose between implementations:</li>
  * <li>If no single tracer service is found, the {@link io.opentracing.NoopTracer NoopTracer} will be used.</li>
  * </ol>
+ *
+ * @see io.opentracing.util.GlobalTracer
+ * @deprecated GlobalTracer is now part of OpenTracing-java
  */
-public final class GlobalTracer implements Tracer {
```

Comment:
Or maybe the simplest approach would be just to add the deprecated annotations on the `get` and `register` methods just documenting the new class?